website/client-old/js/controllers/newGroupCtrl.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+/*
+ A controller to manage the New Group page
+ */
+
+angular.module('habitrpg')
+  .controller("NewGroupCtrl", ['$scope', '$window', 'Groups', 'Payments',
```

Comment:
Sorry could you rephrase?
In the original groupPlansCtrl.js, it had 3 consecutive sections (for search of a better word, each 'section' being hidden/shown using the $scope.activePage). These were BENEFITS -> CREATE_GROUP (where you enter the group details) -> UPGRADE_GROUP (which really means like "pay for right to make groups"). 
The original static page was something completely different (now discarded)
The content of BENEFITS is now shown in the /static/plans and the /#/options/groups/group-plans, using a mixin. But both need to send the user to the inside-the-login CREATE_GROUP page (directly or indirectly) so they can enter the group details and make it. 
This leaves the original groupPlansCtrl.js just a husk, but it's possible /static/plans and /#/options/groups/group-plans deviate in the future so I didn't try to cull it.